README.md

```diff
@@ -105,6 +103,53 @@ React.render(<App person={person} />, document.body)
 person.name = "Mike" // will cause the Observer region to re-render
 ```
 
+In case you are a fan of render props, you can use that instead of children. Be advised, that you cannot use both approaches at once, children have a precedence.
+Example
+
+```javascript
+class App extends React.Component {
+  render() {
+     return (
+         <div>
+            {this.props.person.name}
+            <Observer
+              render={(props) => <div>{props.person.name}</div>}
```

Comment:
`props` work without inejction?

Reply:
Good catch, this is an oversight. It should refer to `this.props` instead.

Reply:
Sorry ,a stupid fault  zzzzzz    should refer to this.props